astroquery/gaia/core.py

```diff
@@ -911,5 +916,24 @@ def launch_job_async(self, query, name=None, output_file=None,
                                         upload_table_name=upload_table_name,
                                         autorun=autorun)
 
+    def get_status_messages(self):
+        """Retrieve the messages to inform users about
+        the status of Gaia TAP
+        """
+        try:
+            subContext = conf.GAIA_MESSAGES
+            connHandler = self._TapPlus__getconnhandler()
+            response = connHandler.execute_tapget(subContext, False)
+            if response.status == 200:
+                if isinstance(response, Iterable):
+                    for line in response:
+                        try:
+                            print(line.decode("utf-8").split('=', 1)[1])
+                        except ValueError as e:
+                            print(e)
+                            pass
+        except OSError:
```

Comment:
It is best to use the most specific error type possible. In this case it looks like `ConnectionError` or perhaps even one of its subclasses would be more appropriate.